content/outcomes/secure-software-supply-chain/building-a-container.md

```diff
@@ -0,0 +1,91 @@
+
+Consider the following Kubernetes pod definition:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kaniko
+spec:
+  containers:
+  - name: kaniko
+    image: gcr.io/kaniko-project/executor:latest
+    args: ["--dockerfile=Dockerfile",
+            "--context=git://github.com/ORG/REPO.git#refs/heads/main",
+            "--destination==<DOCKER USERNAME>/<CONTAINER NAME>"]
+    volumeMounts:
+      - name: kaniko-secret
+        mountPath: /kaniko/.docker
+  restartPolicy: Never
+  volumes:
+    - name: kaniko-secret
+      secret:
+        secretName: regcred
+        items:
+          - key: .dockerconfigjson
+            path: config.json
+```
+
+In this case, a pod will be spun up in Kubernetes using the Kaniko executor image, which will pull down the latest commit in the `main` branch of the GitHub repository defined in the `--context` argument, build it using the Dockerfile in the repository, and upload it to the container registry defined in the `--destination` argument. This build process, and the build process of other similar tools, makes it much easier to integrate a container build into your CI/CD pipelines.
+
+To learn more about Kaniko, please reference the documentation in the [Kaniko GitHub repository](https://github.com/GoogleContainerTools/kaniko).
+
+## Tanzu Build Service
+
+The final step you can take when deciding how to build your container is a fully automated build service such as [Tanzu Build Service](https://tanzu.vmware.com/build-service).  While you can tie tools such as Kaniko into a CI/CD pipeline that you run and manage, tools such as Tanzu Build Service aim to provide that automated build-scan-publish pipeline in a single solution. 
+
+Tanzu Build Service leverages [Cloud Native Buildpacks](/guides/containers/cnb-what-is/), which contain all of the logic on how to build an application, removing the need to write your own Dockerfile. Additionally, Cloud Native Buildpacks provide a standardized base for all of your container images, meaning that if there's a vulnerability found in the Java runtime for example, you only need to update the buildpack. Tanzu Build Service can then take this new buildpack image and rebase your application containers automatically, ensuring these security fixes are implemented across your infrastructure. 
+
+{{< youtube IMmUjUjBzes >}}
+
+If you want to lean more about the technologies that pack Tanzu Build Service, make sure to reference [Getting Started with the pack CLI](/guides/containers/cnb-gs-pack/), a tool that allows you to build containers using Cloud Native Buildpacks locally, as well as [Getting Started with kpack](/guides/containers/cnb-gs-kpack/), the open-source component of Tanzu Build Service that automates the build of containers.
```

Comment:
Perhaps we should add these links to `related` frontmatter?

Reply:
Updated!